src/main/java/jpa/Application.java

```diff
@@ -2,7 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
 
+@EnableJpaAuditing
```

Comment:
JpaAuditing을 통해 중복되는 부분들을 잘 제거해주셨네요 👍

```suggestion
@EnableJpaAuditing(auditorAwareRef = "")
```

아마 알고 계실 것 같은데 이후 `@LastModifiedBy, @CreatedBy`과 같은 부분이 이후 추가되면 auditorAwareRef도 정의해 구현해주셔야 합니다.

관련해서 자세히 알고계신게 아니라면 한번 학습해보셔도 좋을 것 같아요